consumer/src/main/java/com/flipkart/varadhi/consumer/concurrent/CustomThread.java

```diff
@@ -0,0 +1,8 @@
+package com.flipkart.varadhi.consumer.concurrent;
+
+import io.netty.util.concurrent.FastThreadLocalThread;
+
+public class CustomThread extends FastThreadLocalThread {
```

Comment:
whats the purpose of this?

Reply:
Not complete yet. Will be worked on in future, when I start adding core consumption impl. But this thread will used inside an EventExecutor and will allow us to store additional info/fields per thread/EventExecutor.